build/get_new_operator_sdk.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+REL=$(dirname "$0")
+ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
+OS=$(uname | awk '{print tolower($0)}')
+OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.5.0
```

Comment:
I feel like this would be better if you made the version a variable that could be overridden and then used instead of hard coding it across the file.

Reply:
Yes, this sounds better